products/template/build_scripts/load.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Unfortunately, can't comment on empty files. But you can ✂️ the empty __init__, right?

Reply:
hope so, I was swatting down `dcp not found` errors so I'll try without this one